doc/qd-channel.rst

```diff
@@ -55,44 +67,76 @@ into more advanced topics.
 Building New Module
 ===================
 
-Include this subsection only if there are special build instructions or
-platform limitations.
 
-Helpers
-=======
+First, clone the main ns-3 repository, e.g.,:
+
+``git clone https://gitlab.com/nsnam/ns-3-dev ns-3-dev``
+
+Then, clone the qd-channel module into the ``contrib/`` folder:
+
+``git clone https://github.com/signetlabdei/qd-channel ns-3-dev/contrib/qd-channel``
+
+Configure and build ns-3 from the `ns-3-dev` folder:
 
-What helper API will users typically use?  Describe it here.
+| ``./waf configure --enable-tests --enable-examples``
+| ``./waf build``
+
+This module does not provide Python bindings at the moment, they will be added in the near future.
+
+.. Helpers
+.. =======
+
+.. What helper API will users typically use?  Describe it here.
 
 Attributes
 ==========
 
-What classes hold attributes, and what are the key ones worth mentioning?
+* Frequency: a read-only attribute needed for compatibility with ``ns3::ThreeGppSpectrumPropagationLossModel``
```

Comment:
Does a user need to set it?

Reply:
no, it's read-only. should i remove from the doc, then? it's probably unnecessary information..

Reply:
I would comment better on what you mean with read-only - if you call SetAttribute, what happens?

Reply:
Done.
FYI: MakeXxxAccessor does not force you to set both a setter and a getter.
You can
1. point directly to a variable
1. point to a getter/setter pair
1. point to a getter only
1. point to a setter only
See `MakeAccessorHelper` in `attribute-accessor-helper.h`

Reply:
ok - what happens when you call setattribute with an attribute that only has a get?

Reply:
btw you get an error

msg="Attribute name=Frequency is not settable for this object: tid=ns3::QdChannelModel", +0.000000000s -1 file=../src/core/model/object-base.cc, line=197
terminate called without an active exception